packages/react-components/react-tree/src/components/Tree/Tree.cy.tsx

```diff
@@ -0,0 +1,248 @@
+import * as React from 'react';
+import { mount as mountBase } from '@cypress/react';
+import { FluentProvider } from '@fluentui/react-provider';
+import { teamsLightTheme } from '@fluentui/react-theme';
+import {
```

Comment:
**note:** this creates circular dependencies. component-tests approach is the same as for unit-test(jest) test -> import package contents via relative path.